app/assets/javascripts/listings/components/eligibilitySection.js.coffee

```diff
@@ -12,8 +12,10 @@ angular.module('dahlia.components')
       @error = ListingPreferenceService.error
 
       @occupancy = (unitSummary) ->
-        return '1' if unitSummary.maxOccupancy == 1
-        unitSummary.minOccupancy + '-' + unitSummary.maxOccupancy
+        if unitSummary.minOccupancy == 1 || unitSummary.maxOccupancy == null
+          unitSummary.minOccupancy
+        else
```

Comment:
should we put the "at least" in here to make it easier to test and to keep the logic all in the same place?

Reply:
Done!